packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -774,7 +774,7 @@ export class RunningSummarizer implements IDisposable {
 		}
 
 		const result = this.trySummarizeOnce(
-			{ reason: `onDemand/${reason}` },
+			{ summarizeReason: `onDemand/${reason}` },
```

Comment:
Should this be `onDemand;${reason}`?  To match the `SummarizeReason` type.

Reply:
Updated.